cmd/skopeo/proxy.go

```diff
@@ -380,6 +382,53 @@ func (h *proxyHandler) GetManifest(args []interface{}) (replyBuf, error) {
 	return ret, nil
 }
 
+// GetConfig returns a copy of the image configuration, converted to OCI format.
+func (h *proxyHandler) GetConfig(args []interface{}) (replyBuf, error) {
+	h.lock.Lock()
+	defer h.lock.Unlock()
+
+	var ret replyBuf
+
+	if h.sysctx == nil {
+		return ret, fmt.Errorf("Must invoke Initialize")
```

Comment:
Go errors are not capitalized by convention.  It may be helpful to prefix it with `internal error`.

Reply:
OK, fixed that.